addons/website_slides/models/slide_channel_invite.py

```diff
@@ -0,0 +1,137 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import logging
+import re
+
+from email.utils import formataddr
+
+from odoo import api, fields, models, tools, _
+from odoo.exceptions import UserError
+
+_logger = logging.getLogger(__name__)
+
+emails_split = re.compile(r"[;,\n\r]+")
+
+
+class SlideChannelInvite(models.TransientModel):
+    _name = 'slide.channel.invite'
+    _description = 'Channel Invitation Wizard'
+
+    @api.model
+    def _get_default_from(self):
```

Comment:
`_default_email_from` to be guidelines compliant

Reply:
Done.